packages/react-ui/src/theme/sections/components.js

```diff
@@ -588,23 +588,23 @@ export const themeComponentsOverrides = {
 
   MuiListItemIcon: {
     root: {
-      minWidth: spacing(5.75),
-      marginLeft: spacing(0.75),
+      minWidth: getSpacing(5.75),
+      marginLeft: getSpacing(0.75),
 
       '& .MuiSvgIcon-root': {
-        fontSize: spacing(3)
+        fontSize: getSpacing(3)
       }
     }
   },
 
   MuiListItemAvatar: {
     root: {
       '& .MuiAvatar-root': {
-        height: spacing(4.5),
-        width: spacing(4.5)
+        height: getSpacing(4.5),
+        width: getSpacing(4.5)
```

Comment:
This is the type of 'strange' spacing numbers I was talking about... We should consider probably a rounding to the next 'accepted level'. I would suggest creating a new task to discuss with Design about it

Reply:
No, I added it this way to simplify, but you're right. I've just updated this file to be more granular. Let me know if it's ok.

Reply:
Here I just updated the function we use, not the value, but for sure we need to review every spacing we are using to adapt to the new ones that the design team defined. I was thinking of doing it when we start adapting each component.